packages/account/src/Components/forms/__test__/form-select-field.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import FormSelectField from '../form-select-field';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { Formik } from 'formik';
+import userEvent from '@testing-library/user-event';
+
+describe('FormSelectField', () => {
+    const list_items = [
+        { value: '1', text: 'Option 1' },
+        { value: '2', text: 'Option 2' },
+        { value: '3', text: 'Option 3' },
+    ];
+    const mock_store = mockStore({});
+
+    it('should render label', () => {
+        render(
+            <StoreProvider store={mock_store}>
+                <Formik
+                    initialValues={{
+                        test_name: '',
+                    }}
+                    onSubmit={jest.fn()}
+                >
+                    <FormSelectField required label='Test Label' name='test_name' list_items={list_items} />
+                </Formik>
+            </StoreProvider>
+        );
+        expect(screen.getByLabelText('Test Label')).toBeInTheDocument();
+    });
+
+    it('should render and select an option', async () => {
+        render(
+            <StoreProvider store={mock_store}>
+                <Formik
+                    initialValues={{
+                        test_name: '',
+                    }}
+                    onSubmit={jest.fn()}
+                >
+                    <FormSelectField required label='Test Label' name='test_name' list_items={list_items} />
+                </Formik>
+            </StoreProvider>
+        );
+
+        const selectElement = screen.getByLabelText('Test Label');
```

Comment:
```suggestion
        const selected_element = screen.getByLabelText('Test Label');
```
@shahzaib-deriv naming convention dictates to use snake_case for everything that is not a function, or not a constant typed in capitals :)